channels/sl-micro-61-kvm.json

```diff
@@ -0,0 +1,100 @@
+[
+    {
+        "metadata": {
+            "name": "kvm-v2.2.0-4.4-os"
+        },
+        "spec": {
+            "version": "v2.2.0-4.4",
+            "type": "container",
+            "metadata": {
+                "upgradeImage": "registry.suse.com/suse/sl-micro/6.1/kvm-os-container:2.2.0-4.4",
+                "displayName": "SL Micro KVM 6.1 OS",
+                "platforms": ["linux/x86_64","linux/aarch64"]
+            }
+        }
+    },
+    {
+        "metadata": {
+            "name": "kvm-v2.2.0-4.3-os"
+        },
+        "spec": {
+            "version": "v2.2.0-4.3",
+            "type": "container",
+            "metadata": {
+                "upgradeImage": "registry.suse.com/suse/sl-micro/6.1/kvm-os-container:2.2.0-4.3",
+                "displayName": "SL Micro KVM 6.1 OS",
+                "platforms": ["linux/x86_64","linux/aarch64"]
+            }
+        }
+    },
+    {
+        "metadata": {
+            "name": "kvm-v2.2.0-4.1-os"
+        },
+        "spec": {
+            "version": "v2.2.0-4.1",
+            "type": "container",
+            "metadata": {
+                "upgradeImage": "registry.suse.com/suse/sl-micro/6.1/kvm-os-container:2.2.0-4.1",
+                "displayName": "SL Micro KVM 6.1 OS",
+                "platforms": ["linux/x86_64","linux/aarch64"]
+            }
+        }
+    },
+    {
+        "metadata": {
+            "name": "kvm-v2.2.0-4.3-iso"
+        },
+        "spec": {
+            "version": "v2.2.0-4.3",
+            "type": "iso",
+            "metadata": {
+                "uri": "registry.suse.com/suse/sl-micro/6.1/kvm-iso-image:2.2.0-4.3",
+                "displayName": "SL Micro KVM 6.1 ISO",
+                "platforms": ["linux/x86_64","linux/aarch64"]
+            }
+        }
+    },
+    {
+        "metadata": {
+            "name": "kvm-v2.2.0-4.2-iso"
+        },
+        "spec": {
+            "version": "v2.2.0-4.2",
+            "type": "iso",
+            "metadata": {
+                "uri": "registry.suse.com/suse/sl-micro/6.1/kvm-iso-image:2.2.0-4.2",
+                "displayName": "SL Micro KVM 6.1 ISO",
+                "platforms": ["linux/x86_64","linux/aarch64"]
+            }
+        }
+    },
+    {
+        "metadata": {
+            "name": "kvm-v2.2.0-4.1-iso"
+        },
+        "spec": {
+            "version": "v2.2.0-4.1",
+            "type": "iso",
+            "metadata": {
+                "uri": "registry.suse.com/suse/sl-micro/6.1/kvm-iso-image:2.2.0-4.1",
+                "displayName": "SL Micro KVM 6.1 ISO",
+                "platforms": ["linux/x86_64","linux/aarch64"]
+            }
+        }
+    },
+    {
+        "metadata": {
+            "name": "kvm-v2.1.2-2.25-iso"
```

Comment:
There is something wrong here.... it should be v2.2.x :thinking:

I'd say we are missing 91bef7d2 in v1.6 branch 😞 

This is happening because the refresh scripts attempts to keep all minor versions and some images got published on early days before Micro 6.1 release with wrong version and tag....